src/expression_utils.jl

```diff
@@ -22,33 +25,21 @@ end
 ### Parameters/Species/Variables Symbols Correctness Checking ###
 
 # Throws an error when a forbidden symbol is used.
-function forbidden_symbol_check(v)
-    !isempty(intersect(forbidden_symbols_error, v)) &&
-        error("The following symbol(s) are used as species or parameters: " *
-              ((map(s -> "'" * string(s) * "', ",
-                  intersect(forbidden_symbols_error, v))...)) *
-              "this is not permitted.")
-    nothing
-end
-
-# Throws an error when a forbidden variable is used (a forbidden symbol that is not `:t`).
-function forbidden_variable_check(v)
-    !isempty(intersect(forbidden_variables_error, v)) &&
-        error("The following symbol(s) are used as variables: " *
-              ((map(s -> "'" * string(s) * "', ",
-                  intersect(forbidden_variables_error, v))...)) *
-              "this is not permitted.")
```

Comment:
Again, the only time this function is issued it don't actually have any use. Old remnant and thus removed.